test/integration/shebang/src/cases/cjsWithShebang.cjs

```diff
@@ -0,0 +1,3 @@
+#!/usr/env node
+
+module.exports = 789
```

Comment:
Lets rename this test fixture to `hashbang` to match the name of the spec https://github.com/tc39/proposal-hashbang